staff/sergio-luz/cv-styled/index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Jaime Gonzalez: curriculum vitae</title>
+</head>
+
+<body class="body__grid-container">
+    <header class="element--Bckcolor element--border text--center">
+        <h1>Jaime Gonzalez / Web Developer</h1>
+        <address>Gran via de les Corts Catalanes 587</address>
```

Comment:
Why did you create two `<address>` and  a `<a>` tag as brothers instead of wraping them in a single `<address>` tag?

Consider the following example from MDN:
```
<address>
  James Rockford<br/>
  2354 Pacific Coast Highway<br/>
  California<br/>
  USA<br/>
  +311-555-2368<br/>
  Email: <a href="mailto:j.rockford@example.com">j.rockford@example.com</a><br/>
</address>
```
Further information:
- [The Contact Address element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address)